### ISSUE TITLE
Unable to move AMP story block using keyboard

### ISSUE THREAD

**Issue by spacedmonkey** [ampproject/amp-wp#3325]
## Bug Description

When using keyboard navigation, users are unable to move movable blocks in the AMP Story interface. For accessible reasons, this should be possible. 

## Additional context

 - WordPress version:
 - Plugin version:
 - Gutenberg plugin version (if applicable):
 - AMP plugin template mode:
 - PHP version:
 - OS:
 - Browser: [e.g. chrome, safari]

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* After selecting a block, I should be able to move the block up, down, left and right via the direction keys. (QUESTION: HOW MUCH SHOULD IT MOVE ON EACH KEY STROKE?)

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

**Comment by barklund** [comment]
It's not trivial, but here's a [great guide to best practice expected behaviour](https://www.w3.org/wiki/PF/ARIA/BestPractices/DragDrop).

**Comment by swissspidy** [comment]
Previously suggested in #2454. Will close it as a duplicate.

**Comment by spacedmonkey** [comment]
As a talking point, I have created a WIP pull request in #3400. It is still discovery stages at this point, but it is a workable solution.

**Comment by spacedmonkey** [comment]
Screencast of the tabbing and arrows in use. 
https://www.youtube.com/watch?v=u280hGEgCd0&feature=youtu.be

**Comment by spacedmonkey** [comment]
@MackenzieHartung @pbakaus A lot of work and discussion has taken place on #3400 . This ticket is not just a simple bug, it has lots of moving parts and is a fundamental change on how blocks are moved. This PR will effect other on going such as #3060 #3311  #3416 . This ticket needs to go through the new process, so we can capture all requirements

**Comment by jauyong** [comment]
Hey @jasti I only added the one AC for now. Based on @spacedmonkey's comments I'm not sure if that is too high level and if more is needed. I will wait for you to move this to the IB column...

**Comment by swissspidy** [comment]
In #3400 I left a comment for a new feature that could be its own AC:

> It would be nice if one could move in bigger steps when holding down an additional key, e.g. ctrl, alt, or cmd. If I am not mistaken this kind of behavior is common in other applications when moving elements with the keyboard.

So just arrow keys: move by single pixels
Meta key + arrow key: move in steps of N pixels
